user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_updates.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: Query_updates
+---
+
+# Query updates
+
+When executing a query, you can enable query updates to keep your query result up to date.
+
+These updates may add, remove, or update rows from the initial result returned by your query.
+
+> [!TIP]
+> To see which updates a query can support, refer to [query update support](#query-update-support).
+
+## Enabling updates
+
+While GQI allows updates to be enabled or disabled for each query individually, in the UI, this is configured per component.
```

Comment:
Do we need to mention that GQI allows this per query? Unless there is a way a user could configure this per query, I'm worried this could lead to unnecessary questions or confusion.

Reply:
I think it gives a better mental model to the user of how the queries actually work, and likely can prevent confusion of why things behave the way they do.
Plus, it will make more sense in case they get errors or when we evolve the feature later on.